.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 tmp
+*.swp
```

Comment:
Since this line has nothing to do with this feature, please remove it here, personally, I'll respectfully suggest you to put it in git global ignore list like this:

https://github.com/PeterDaveHello/Unitial/blob/c779eac4187930b9d780fecfaadac23ff0a9f02b/gitignore_global#L2
https://github.com/PeterDaveHello/Unitial/blob/c779eac4187930b9d780fecfaadac23ff0a9f02b/gitconfig#L69